### ISSUE TITLE
subscriptionType support

### ISSUE THREAD

**Issue by rukor** [walmartlabs/lacinia#25]
When I try to use [graphql-ide](https://github.com/redound/graphql-ide) it fails with this error message:

```
{"errors":[{"message":"Cannot query field `subscriptionType' on type `__Schema'.","query-path":["__schema"],"locations":[{"line":3,"column":13}],"field":"subscriptionType","type":"__Schema"}]}
```

Any ideas?

**Comment by annapawlicka** [comment]
Lacinia doesn't support subscriptions at the moment. It only supports queries and mutations.

**Comment by bcarrell** [comment]
So I think you guys are talking about two different things.

- Lacinia doesn't support subscriptions because Lacinia follows the specification and the specification doesn't have subscriptions yet.
- `graphiql` seems to do some funky stuff where it issues a query with subscriptions first to see what it's dealing with and then is supposed to issue a [query without subscriptions](https://github.com/graphql/graphiql/blob/dc4a561f796258c6e6773612279abccb76f10ee0/src/utility/introspectionQueries.js#L14) when that fails.  Locally, we've seen errors similar to yours, but we have workarounds for it.  So, I'm not sure why that part's not working.

tl;dr We'll add some more documentation on how to setup graphiql properly (or make a separate project with it working out of the box).

**Comment by rukor** [comment]
Thanks 

I found this out while googling and then changed graphql-ide to use a custom introspection query that does not have subscriptions. Please kindly close this issue.


------------
Roland Ukor
